### DISCUSSION TITLE
Hue Scenes in Bubble Cards

### DISCUSSION THREAD

**Discussion by komtx** [Clooos/Bubble-Card#466] [General]
Hello all.  Still learning HA, but know enough to be dangerous now.  Love the new bubble cards, but I am having a hard time getting this straight in my brain.  I have 15 zones of light (all Hue bulbs (sorry)), and each of those sets have their own scenes.  Trying to figure out how to make a bubble card that would handle these in a useful, user friendly manner, and I'm striking out.

Appreciate any guidance and thanks!

**Comment by paulsgale** [comment]
I have Hue everywhere. The whole house is made up of a couple hundred Hue devices and three bridges!
Hue scenes are stateless (or rather the way the integration works is stateless) which means you can’t easily track when a scene is on and hence display an on state when you turn a scene on. We can trigger a scene from a button though but I’m not aware of anything in bubble card specifically that shows the state of the scene ie shows the button as ON. 

I think @Clooos has Hue too so would be interesting to see if he has a solution to this. There has been a bit of discussion on the HA forum about scenes and HA’s lacklustre implementation of scenes.

**Reply by Clooos** [reply]
Indeed I'm almost full Hue too, but I'm not using the scenes, I've made multiple scripts that change the lights based on an input_select, with this it's easy to know what scene/script is on. 

But I haven't took the time to integrate it on my Bubble Card dashboard, they are automatically changing based on my automations, and I can control them with Google Assistant. It wouldn't be that hard to make some buttons with sub-buttons to control that.

**Reply by pgale** [reply]
It's fairly simple to track if a scene is on or off *IF* you control access to Hue and only expose scenes via dimmer switches and from HA. If you then have individual lights changing at any time, of course the scene isn't then on and that's much more difficult to track. You'd have to constantly check for light status in each scene I guess. 

I use scenes pretty much exclusively as I have larger numbers of lamps and hue devices in most rooms. Controlling them via a scene is a) much easier b) easy to setup in the Hue app c) works smoothly and SIMULTANEOUSLY sets each device to the colour and level. 

Using individual controls or using scenes in HA of course doesn't do this. The Hue bridge is designed to give this smooth and simultaneous operation as you know - that's why I use the Hue bridges still. Others have migrated all devices to HA but for me, that's a big backward step.

**Reply by Clooos** [reply]
This is simultaneous and smooth with my scripts, never had a single issue. For me it's better to control everything inside of Home Assistant, I never use any of the Hue services nor the app.

**Reply by pgale** [reply]
That's very odd. People on other threads on the forum have said it isn't simultaneous from the Hue Bridge. Do you have your devices directly connected to HA?

**Reply by Clooos** [reply]
Maybe it isn't simultaneous but I've never seen a difference. I've just made groups directly in the Hue app.

**Reply by komtx** [reply]
Thanks both!  I have about 100 bulbs and two bridges and also use scenes as well as alarms for my kids.  My goal is to one day have all scenes automated (I did just make an automation to turn on to a Daylight color temp downstairs if cloudy/rainy), but for now, I just toggle them via Dashboards.  Wife and kids use dimmer switches so not an issue to restrict all Hue activity to either HA or Switches.  

Until I found Bubble cards (I'm still new to HA :)), I used an entity card for the bulbs and scenes.  After seeing Bubble, I think having the individual bulbs is not necessary, or perhaps have one dashboard for Admin that has them all listed in case needed.  Ideally, I'd love to be able to click on the icon and have the Hue Scenes pop up - see second pic which is my first attempt at using Bubble.  
![FireShot Capture 027 - Rick – Home Assistant - homeassistant local](https://github.com/Clooos/Bubble-Card/assets/92494560/4e484188-203f-4fc1-ac35-8d356159eb22)
![FireShot Capture 028 - Rick – Home Assistant - homeassistant local](https://github.com/Clooos/Bubble-Card/assets/92494560/8337e9ce-078c-47af-a4d8-a5b2e3b8ec9d)
